libs/pageserver_api/src/config.rs

```diff
@@ -284,7 +285,6 @@ pub struct DiskUsageEvictionTaskConfig {
     /// Select sorting for evicted layers
     #[serde(default)]
     pub eviction_order: EvictionOrder,
-    #[serde(default = "default_true")]
```

Comment:
wait this disables disk usage based eviction by default.

Reply:
No, it will be taken from `DiskUsageEvictionTaskConfig::default()`